Lib/multiprocessing/managers.py

```diff
@@ -1262,8 +1262,12 @@ class SharedMemoryServer(Server):
 
         def __init__(self, *args, **kwargs):
             Server.__init__(self, *args, **kwargs)
+            address = self.address
+            # The address of Linux abstract namespaces can be bytes
+            if isinstance(address, bytes):
+                address = address.decode()
```

Comment:
Maybe use os.fsdecode() which escapes non-decodable characters as surrgate characters.

Reply:
👍

Reply:
Does it mean that you like os.fsdecode()? But you didn't modify your code to use it?

Reply:
It means that I forgot 😟

Reply:
Done in my last commit